### PR TITLE
Upstream master

### PR DESCRIPTION
**Pull request by PetrDlouhy**
second set of patches with czech localization + one bug fix

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
 #
 
 class User < ActiveRecord::Base
-  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled
+  attr_accessible :email, :full_name, :display_name, :password, :password_confirmation, :disabled, :role, :profile_attributes
```
Comment (nikolai-b) [inline app/models/user.rb]: hi, could you explain why you've added `role` and `profile_attributes` here?  Maybe I'm missing something but I can't see them being used.

Comment (nikolai-b) [inline app/models/user.rb]: Just seen this is related to a bug.  Could you explain the bug a bit?  I couldn't see an obvious related open issue.

Comment (gravitystorm) [inline app/models/user.rb]: We certainly don't want role to be mass-assignable, that's a security hole.

**Comment by PetrDlouhy** [comment]
I created an issue to address problem that I had. I also changed the original commit, so now it allows only admin to change that attributes. I am not Ruby programmer, but I hope this time it will be correct.

**Comment by nikolai-b** [comment]
Thanks very much for the bug report and the translations. Sadly I think the permissions still aren't right, I've fixed them [on this branch](https://github.com/cyclestreets/cyclescape/commit/deeee2b89c3a5fa336bf07f0d650c5e972a90782).

Would you mind removing that commit from this pull request and then I'll merge in the translation fixes.

**Comment by mvl22** [comment]
In order to get this moving forward, can I suggest that this PR be closed, and that a new one, simply the contents of
https://github.com/auto-mat/toolkit/commit/f21173305363af64fe71e28fb0cb28806d61331b
is made, and that merged straight into staging then master? It seems straightforward.

**Comment by nikolai-b** [comment]
Thanks @PetrDlouhy, I've included your commit e600ed0

**Pull request closed without merging**